kernel-headers/CMakeLists.txt

```diff
@@ -6,12 +6,12 @@ publish_internal_headers(rdma
   rdma/i40iw-abi.h
   rdma/ib_user_cm.h
   rdma/ib_user_ioctl_cmds.h
-  rdma/ib_user_ioctl_verbs.h
   rdma/ib_user_mad.h
```

Comment:
I'm not sure the header should be removed from the rdma/ directory.. It should be OK to have it both places and is more overall consistent.